.github/workflows/pkg-js-release.yaml

```diff
@@ -0,0 +1,29 @@
+name: Release (JS)
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "pkg/js/v*"
+
+jobs:
+  test:
+    uses: ./.github/workflows/pkg-js-build.yaml
+    secrets: inherit
+
+  release:
+    runs-on: ubuntu-latest
+    needs: [test]
+    permissions:
+      contents: write
+      packages: write # publish a new github release
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+        with:
+          fetch-depth: 0
+
+      - uses: Roang-zero1/github-create-release-action@57eb9bdce7a964e48788b9e78b5ac766cb684803 # v3.0.1
```

Comment:
Can you add the part for publishing to npm here? (see the JS SDK for an example)